pages/Sidebar.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import Link from 'next/link';
+
+const Sidebar = () => {
+  const [selectedItem, setSelectedItem] = useState('About');
+
+  const handleItemClick = (item) => {
+    setSelectedItem(item);
+  };
+
+  return (
+    <div className='fixed top-0 left-0 h-screen w-1/5 box-border bg-[#081f2e] '>
+      <div className='flex-row h-70 text-center my-4'>
+        <img src='/images/joemon1.jpg' alt="Image Description" className="w-[10rem] h-[10rem] mx-auto rounded-full " />
+        <h3>Joemon Mathew</h3>
+      </div>
+      <ul className='flex flex-col items-center h-full  mr-1 space-y-5 bg-[#020107] box-border '>
+        <li className='mt-5'>
+          <Link href='/About'
+            onClick={() => handleItemClick('About')}
```

Comment:
You don't need to keep a state to manage the selection. Instead, you can check the current path and style the element accordingly. Check this out: https://stackoverflow.com/a/59410881/5780058